boa/vm/fork.py

```diff
@@ -277,31 +278,31 @@ def commit(self, checkpoint):
 
     def record(self):
         checkpoint = super().record()
-        self._dontfetch.record(checkpoint)
-        return checkpoint
+        return self._dontfetch.record(checkpoint)
```

Comment:
is this change necessary? i wouldn't trust that dontfetch.record() returns the same thing as super.record()

Reply:
`record` must return the checkpoint, and since you've given a custom one it must respect it.
https://github.com/ethereum/py-evm/blob/d8df507e42c885ddeca2b64bbb8f10705f075d3e/eth/db/journal.py#L143

But yeah it's not necessary, I can revert it

Reply:
i mean i'm aware of what the implementation *currently* does, but if it ever changes, we want to match super, not dontfetch.